index.html

```diff
@@ -17,8 +17,10 @@ <h1>Imagegram</h1>
     <div class="container">
     </div>
     <div class="lightbox">
```

Comment:
These html elements (div.lightbox, etc.) could also have been added in the main.js file with javascript.